examples/terraform/aws/output.tf

```diff
@@ -72,7 +72,9 @@ output "kubeone_workers" {
           diskSize         = 50
           diskType         = "gp2"
           ## Only applicable if diskType = io1
-          diskIops = 500
+          diskIops         = 500
+          isSpotInstance   = false
+          ebsVolumeEncrypted = false
```

Comment:
@kron4eg Those are newly added API fields (`isSpotInstance` is older, but we added it just now). Should both be false?

Reply:
yes, and also what will be required to use `ebsVolumeEncrypted=true`? I guess some AWS KMS magic whatever, but still, question is open.

Reply:
I'm not really sure. I have assumed the same (regarding AWS KMS magic) and that's the reason I set it to false.